jax_tqdm/pbar.py

```diff
@@ -131,29 +131,29 @@ def _define_tqdm(arg, transform):
         tqdm_bars[0].set_description(message, refresh=False)
 
     def _update_tqdm(arg, transform):
-        tqdm_bars[0].update(arg)
+        tqdm_bars[0].update(int(arg))
```

Comment:
Just for my own understanding, this is casting what is now an `jax.Array` into a Python `int`. In the issue you mentioned `jax.tree.map(np.asarray, args)` but I guess here we only need to cast this single value?

Reply:
Also this should be backward compatible with older versions of jax right?

Reply:
Yes - jax wraps arguments to callback as arrays now, so this change casts the the argument to `_update_tqdm` (which is either `print_rate` or `remainder`) back to an `int` which `tqdm` can parse. And yes, since there is only one argument in this case I just used `int(arg)` directly.

This should be backwards compatible according to https://jax.readthedocs.io/en/latest/changelog.html#jax-0-4-27-may-7-2024.